### DISCUSSION TITLE
Undraft PR

### DISCUSSION THREAD

**Discussion by kanimaru** [community/community#41631] [Apps, API and Webhooks]
### Select Topic Area

Show & Tell

### Body

Hi how can you undraft a PR via REST or Graphql didn't find a solution in the docs.

**Comment by spenserblack** [comment]
The [GitHub CLI](https://github.com/cli/cli) does exactly that with `gh pr ready`, so that could be a good place to start looking for examples.

**Reply by kanimaru** [reply]
I looked into it and used it as base 
```graphql
mutation MarkPullRequestReadyForReview($pullRequestId: ID!) {
  markPullRequestReadyForReview(input: {
    pullRequestId: $pullRequestId
  }) {
    clientMutationId
  }
}
```

but now I get:
```graphql
GraphQLResponse{
data={"markPullRequestReadyForReview":null}, 
errors=[
  Error{
    message=Resource not accessible by integration, 
    locations=[{line=2, column=3}], 
    path=[markPullRequestReadyForReview], 
    extensions={saml_failure=false}, 
    otherFields={type=FORBIDDEN}}
]
}
```
It's an Github Application installed for the repository with following permissions:
- PR Read/Write
- Issue Read/Write
- Contents Read/Write access

Could it be thats not possible for Github Application?
Didn't find anything in the docs to it :(

**Reply by spenserblack** [reply]
:thinking: I'd have to look into it to give you a proper answer, but you might get a faster response raising a Q&A discussion at cli/cli. I'm sure either one of the maintainers or a contributor who is more familiar will be happy to help.
